README.md

```diff
@@ -46,6 +51,43 @@ pip to install directly from the master branch on github:
 pip install git+https://github.com/keras-team/keras-cv.git tensorflow --upgrade
 ```
 
+## Using KerasCV with Keras Core
+
+As of version 0.6.0, KerasCV supports multiple backends with Keras Core out of 
+the box. There are two ways to configure KerasCV to run with multi-backend 
+support:
+
+1. Via the `KERAS_BACKEND` environment variable. If set, then KerasCV will be 
+using Keras Core with the backend specified (e.g., `KERAS_BACKEND=jax`).
+2. Via the `.keras/keras.json` and `.keras/keras_cv.json` config files (which 
+are automatically created the first time you import KerasCV):
+- Set your backend of choice in `.keras/keras.json`; e.g., `"backend": "jax"`. 
```

Comment:
If you indent this two more space, does it align under the number when rendered? Would be nice if we could make it visually clear this is all part of `2.`

Reply:
Not obviously, but this is the right syntax